content/applications/inventory_and_mrp/manufacturing/plm/version_control.rst

```diff
@@ -0,0 +1,138 @@
+===============
+Version control
+===============
+
+.. |BOM| replace:: :abbr:`BoM (Bill of Materials)`
+.. |BOMs| replace:: :abbr:`BoMs (Bills of Materials)`
+.. |ECO| replace:: :abbr:`ECO (Engineering Change Order)`
+.. |ECOs| replace:: :abbr:`ECOs (Engineering Change Orders)`
+
+Odoo's Product Lifecycle Management (PLM) app helps to easily manage the revision history of bills
+of materials (BoMs). :menuselection:`PLM` can be used to access previous |BOM| versions, assembly
+instructions, component details, and past product design files. |BOM| effective dates are recorded
+for error tracking, and changes are compartmentalized in an :ref:`engineering change order
+<plm/eco>`, which makes merging |ECOs| back to the production |BOM| simple.
+
+Current BoM version
+===================
+
+To see the current version of the |BOM| used in production, go to :menuselection:`PLM app --> Master
+Data --> Bill of Materials`, and select the desired |BOM| from the list. Then, switch to the
+:guilabel:`Miscellaneous` tab, where the currently live :guilabel:`Version` of the |BOM| is
+displayed.
+
+.. note::
+   |BOMs| can also be accessed from :menuselection:`Manufacturing app --> Products --> Bill of
+   Materials`.
+
+Version history
+===============
+
+To view all |ECOs| linked to a |BOM|, click the :guilabel:`ECO` smart button on a |BOM|. Click the
+:guilabel:`≣ (four horizontal lines)` icon to switch to list view.
```

Comment:
Does every BoM have this ECO smart button, even if there are 0 past ECOs for the BoM? Might be helpful to include a quick sentence about that